config/jobs/kubernetes-sigs/jobset/jobset-presubmit-release-0.4.yaml

```diff
@@ -0,0 +1,174 @@
+presubmits:
+  kubernetes-sigs/jobset:
+  - name: pull-jobset-test-unit-release-0-4
+    cluster: eks-prow-build-cluster
+    branches:
+    - ^release-0.4
+    skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$"
+    decorate: true
+    path_alias: sigs.k8s.io/jobset
+    annotations:
+      testgrid-dashboards: sig-apps
+      testgrid-tab-name: pull-jobset-test-unit-release-0-4
+      description: "Run jobset unit tests"
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240409-13cd3acf7e-master
+        env:
+        - name: GO_TEST_FLAGS
+          value: "-race -count 3"
+        command:
+        # generic runner script, handles DIND, bazelrc for caching, etc.
+        - runner.sh
+        args:
+        - make
+        - test
+        securityContext:
+          privileged: true
+        resources:
+          limits:
+            cpu: 3
+            memory: 10Gi
+          requests:
+            cpu: 3
+            memory: 10Gi
+  - name: pull-jobset-test-integration-release-0-4
+    cluster: eks-prow-build-cluster
+    branches:
+    - ^release-0.4
+    skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$"
+    decorate: true
+    path_alias: sigs.k8s.io/jobset
+    annotations:
+      testgrid-dashboards: sig-apps
+      testgrid-tab-name: pull-jobset-test-integration-release-0-4
+      description: "Run jobset integration tests"
+    spec:
+      containers:
+      - image: public.ecr.aws/docker/library/golang:1.22
+        command:
+        - make
+        args:
+        - test-integration
+        resources:
+          limits:
+            cpu: 3
+            memory: 10Gi
+          requests:
+            cpu: 3
+            memory: 10Gi
+  - name: pull-jobset-test-e2e-release-0-4-1-27
+    cluster: eks-prow-build-cluster
+    branches:
+    - ^release-0.4
+    skip_if_only_changed: "^docs/|^\\.github/|\\.(md)$|^(README|LICENSE|OWNERS)$"
+    decorate: true
+    path_alias: sigs.k8s.io/jobset
+    annotations:
+      testgrid-dashboards: sig-apps
+      estgrid-tab-name: pull-jobset-test-e2e-release-0-4-1-27
+      description: "Run jobset end to end tests for Kubernetes 1.27"
+    labels:
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240409-13cd3acf7e-master
+        env:
+        - name: E2E_KIND_VERSION
+          value: kindest/node:v1.27.3
+        - name: BUILDER_IMAGE
+          value: public.ecr.aws/docker/library/golang:1.21
+        command:
+        # generic runner script, handles DIND, bazelrc for caching, etc.
+        - runner.sh
+        args:
+        - make
+        - kind-image-build
+        - test-e2e
```

Comment:
In JobSet the make rule for e2e tests is [test-e2e-kind](https://github.com/kubernetes-sigs/jobset/blob/947df5da269dadd66a3c819497983834a834f586/Makefile#L27). Please check the various make commands in this PR to ensure they are using the JobSet make rules and not the Kueue ones.